examples/chef/common/clusters/media-playback/MediaPlaybackManager.cpp

```diff
@@ -99,10 +114,34 @@ CHIP_ERROR MediaPlaybackManager::HandleGetAvailableTextTracks(AttributeValueEnco
     });
 }
 
+CHIP_ERROR MediaPlaybackManager::HandleSetCurrentState(chip::app::Clusters::MediaPlayback::PlaybackStateEnum currentState)
+{
+    Status status = Attributes::CurrentState::Set(mEndpoint, currentState);
+
+    if (Status::Success != status)
+    {
+        ChipLogError(Zcl, "Unable to save CurrentState attribute, 0x%x", to_underlying(status));
+    }
+
+    return CHIP_NO_ERROR;
```

Comment:
Shouldn't this in fact return `CHIP_ERROR_IM_GLOBAL_STATUS_VALUE(status)`?

Reply:
Fixed done. Thanks for the correction.